synapse/models/inet.py

```diff
@@ -1489,6 +1489,16 @@ def getModelDefs(self):
                     ('inet:ssl:jarmsample', ('comp', {'fields': (('server', 'inet:server'), ('jarmhash', 'inet:ssl:jarmhash'))}), {
                         'doc': 'A JARM hash sample taken from a server.'}),
 
+                    # TODO: Form Inheritance to hash:md5
```

Comment:
Do we want to block on https://github.com/vertexproject/synapse/pull/3617 for this, leave it as a guid, or loosen to an md5 compatible string  that copies `hash:md5`?